app/Controllers/CommentControllers.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controllers;
+
+use App\View;
+use App\Models\Post;
+use App\Models\User;
+use App\Models\Comment;
+
+class CommentControllers
+{
+
+    protected $commentModel;
```

Comment:
Generate a PHPDoc block for a code construct
https://www.jetbrains.com/help/phpstorm/phpdoc-comments.html

![Screenshot from 2022-06-22 16-19-29](https://user-images.githubusercontent.com/100173186/174962586-90d34093-82e3-4dae-8d84-258e751ba274.png)

Reply:
should this be done on every function? Its not difficult to implement, but just want to practice getting coding standard correct.